### PR TITLE
CoreDNS chart: update to latest version

### PR DESCRIPTION
**Pull request by miekg**
Update CoreDNS' chart to version 0.9.10 and replace occurences of
middleware with plugin as this is also used upstream.

### CODE REVIEW

**Comment by unguiculus** [comment]
Bump for CLA check.

**Comment by unguiculus** [comment]
Would you like to update the versions again before merging?

**Comment by miekg** [comment]
Ack. But note I'm releasing CoreDNS 1.0.1 tomorrow or Monday :)
(also see in error in this PR - cidr option is gone in the k8s plugin)

File: stable/coredns/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 name: coredns
-version: 0.7.0
-description: CoreDNS is a DNS server that chains middleware and provides Kubernetes
-  DNS Services
+version: 1.0.0
```
Comment (unguiculus) [inline stable/coredns/Chart.yaml]: Note that there's `version` and `appVersion`. The latter is still missing. Please add it.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Didn't you want to go for 1.0.1?

File: stable/coredns/Chart.yaml (hunk 1 above)
Comment (miekg) [reply stable/coredns/Chart.yaml]: Done. assuming Version is the Chart's version: bumbed that to 0.8.0 (instead of 1.0.0).

**Review by unguiculus — APPROVED** [review]

**Pull request merged**